### ISSUE TITLE
README has outdated output log example

### ISSUE THREAD

**Issue by jhoshiko** [NREL/hive#156]
In the [README](https://github.com/NREL/hive/blob/5cc27d7e463865c15e556e85c989338e38af84e1/README.md?plain=1#L154) file, the example output log is outdated. We should update this log with a fresh Denver demo run.

**Comment by nreinicke** [comment]
yeah good call; I wonder if this could even become automated by moving this log over to the documentation and then running the denver demo whenever we build the docs.

**Comment by clintonsteiner** [comment]
Opened a PR here, https://github.com/NREL/hive/pull/160 I'm at pycon and will be around tomorrow
First PR, hope I did this correctly

**Comment by clintonsteiner** [comment]
Readme updated, automating this still needs to be done with github actions
